iamy/aws.go

```diff
@@ -120,8 +120,11 @@ func (a *AwsFetcher) fetchIamData() error {
 			}),
 		},
 		func(resp *iam.GetAccountAuthorizationDetailsOutput, lastPage bool) bool {
+			// There's a bug here. This doesn't set the variable outside this function scope
+			// So IAMY just swallows any errors returned from populateIamData()
```

Comment:
hmm I'm not sure I understand... can you fix or produce a testcase?

Reply:
I tried to fix it, but I'm not entirely sure. I'll pull this out into a seperate WIP PR and see if I can write a failing test.